Cargo.toml

```diff
@@ -35,5 +36,7 @@ targets = [
     "x86_64-apple-darwin",
     "x86_64-pc-windows-msvc",
     "x86_64-unknown-linux-gnu",
+    "wasm32-unknown-unknown",
+    "wasm32-wasi",
```

Comment:
You can delete all the other targets; this is what ProxyWasm supports amd the user guide recommends

Reply:
Done.